CHANGES.rst

```diff
@@ -202,8 +202,6 @@ Bug Fixes
     ``median_absolute_deviation``. And allow to use these functions with
     a multi-dimensional ``axis``. [#5835]
 
-- ``astropy.sphinx``
-
```

Comment:
@keflavich You inadvertently deleted these lines instead of the ones below (line 551).